src/Grid/Grid.js

```diff
@@ -1,12 +1,84 @@
+import Paper from 'material-ui/Paper';
+import PropTypes from 'prop-types';
 import React from 'react';
+import { withStyles } from 'material-ui/styles';
+import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    marginTop: theme.spacing.unit * 3,
+    overflowX: 'auto'
+  }
+});
 
 class Grid extends React.Component{
+  static defaultProps = {
+    rowsPerPage: 5,
+    page: 0,
+    initialSort: {
+      column: '',
+      order: 'asc'
+    },
+    title: ''
+  }
+
+  state = {
+    order: this.props.initialSort.order,
+    orderBy: this.props.initialSort.column,
+    page: this.props.page,
+    rowsPerPage: this.props.rowsPerPage,
+    data: this.props.data
+  }
+
   render() {
+    const { classes, columns } = this.props;
+    const { data, rowsPerPage, page, order, orderBy } = this.state;
     return (
-      <div style={{ textAlign: 'center' }}>
-        <h1>Hello World</h1>
-      </div>);
+      <Paper className={classes.root}>
+        <Table className={classes.table}>
+          <TableHead>
+            <TableRow>
+              { columns.map(column => 
+                <TableCell key={column.key}>
+                  {column.label}
+                </TableCell>
+              )}
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {
+              data.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row, rowIndex) => (
+                <TableRow hover key={rowIndex}>
+                  { columns.map((column, index) => 
+                    <TableCell key={index} padding={column.label === '' ? 'none' : 'default'}>
+                      { row[column.key] && row[column.key] }
+                    </TableCell>
+                  ) }
+                </TableRow>
+              ))}
+          </TableBody>
+        </Table>
+      </Paper>);
   }
 }
 
-export default Grid;
\ No newline at end of file
+Grid.propTypes = {
+  classes: PropTypes.object.isRequired,
+  columns: PropTypes.arrayOf(
```

Comment:
To keep the implementation consistent, let's use this column model: https://github.com/unosquare/tubular/blob/master/src/js/tubular/models/tubular-column.js#L32